packages/http/lib/http.tsp

```diff
@@ -21,16 +21,16 @@ model Response<Status> {
 /**
  * Defines a model with a single property of the given type, marked with `@body`.
  *
- * This can be useful in situations where you cannot use a bare T as the body
+ * This can be useful in situations where you cannot use a bare type as the body
  * and it is awkward to add a property.
  *
- * @template T The type of the model's `body` property.
+ * @template TBody The type of the model's `body` property.
  */
 @doc("")
-model Body<T> {
+model Body<TBody> {
```

Comment:
Would `Type` would be better or something where we don't add the `T` prefix as this seems to be the pattern you have done